activiti-engine/pom.xml

```diff
@@ -156,6 +160,21 @@
       <artifactId>hibernate-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>batik</groupId>
+      <artifactId>batik-svg-dom</artifactId>
+      <scope>test</scope>
+    </dependency>
+	<dependency>
+	  <groupId>batik</groupId>
+	  <artifactId>batik-anim</artifactId>
+	  <scope>test</scope>
+	</dependency>
+	<dependency>
+	    <groupId>batik</groupId>
+	    <artifactId>batik-parser</artifactId>
+	    <scope>test</scope>
+	</dependency>
```

Comment:
@daisuke-yoshimoto I can see in other classes inside the engine where we call the image generator helper, but is it possible to move all the logic to the image generator project? Even the tests should go into that image generator project. I will add more comments in the places where we can just remove the responsibility of the engine of knowing about how to generate images.